easybuild/easyconfigs/r/R/R-3.5.1-foss-2018b-Python-2.7.15.eb

```diff
@@ -1951,6 +1951,7 @@ exts_list = [
         'checksums': ['58e36d3b8a67d01194c4e5099499657d017f77fc2081213633964b1d57a7e32f'],
     }),
     ('hdf5r', '1.0.0', {
+        'installopts': '--configure-args="--with-hdf5=$EBROOTHDF5/bin/h5pcc"',
```

Comment:
@smoors What goes wrong if this is not used?

Reply:
if there is a system HDF5 present, `hdf5r` will use that one.

Reply:
...and if the system HDF5 is too old, build will fail.